src/kaleidoscope/driver/storage/Flash.h

```diff
@@ -0,0 +1,80 @@
+/* -*- mode: c++ -*-
```

Comment:
instead of calling it "Flash" can we call it something more descriptive if what it really is?

Alternatively, since it's using the emulated EEPROM library, couldn't we just use the EEPROM driver?

Reply:
We could, with a few tricks: we'd need to include a different header on SAMD, and `EEPROM.commit()` doesn't exist on AVR, so that'd need an `#ifdef __SAMD__`. The reason I didn't do that, is because eventually I want to move towards using `FlashAsStorage` _without_ the EEPROM API emulation. If we don't bind the driver to the `EEPROM` name, then this becomes easier: just replace the implementation.

Reply:
I guess I'm ok with it as it is, but I do feel like Flash is a really, really generic name that's going to bite us down the road.